### DISCUSSION TITLE
using python 3.12 over 3.8?  advantages and disadvantages?

### DISCUSSION THREAD

**Discussion by l-o-u-i-s-g** [beeware/toga#2624] [General]
I was using python 3.12  until  I required  the cryptography package which is not supported by python 3.12  for chaquo (android).
 
So now the question that comes to mind is should I just create all my projects  using python 3.8 for now?  
Am I missing something by not using python 3.12?  Performance differences?  etc ...  
Maybe I'll encounter issues later with other packages only supported by 3.12?  
Is it just generally better to use python 3.8 for now?

**Comment by mhsmith** [comment]
For now, Python 3.8 has the largest set of packages, so it's probably best for you to use that. We're working on improving the situation for later Python versions, but the results of that won't become available until later in the year.

Later Python versions do have better performance, but you're unlikely to notice the difference in most apps.
